recipe/meta.yaml

```diff
@@ -15,16 +15,20 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python 3.8
```

Comment:
You should use `{{ python_min }}` here and below. If there are very specific reasons for still supporting python 3.8 (already end-of-life for almost 2 years), then you can override the default value at the top of the recipe, e.g.
```
{% set python_min = "3.8" %}
```

More details can be found [here](https://github.com/conda-forge/cfep/blob/main/cfep-25.md)